environments/stg/backend_lb_config.yaml

```diff
@@ -55,4 +55,8 @@ gateways:
       # DTS Legacy Apps
       - product: lgy-iac
         component: web
-        ssl_enabled: true
\ No newline at end of file
+        ssl_enabled: true
+      # PET team
+      - product: c100
+        component: c100-application
```

Comment:
You'll need to set this to application otherwise the dns record will be c100-c100-application